build.gradle

```diff
@@ -1,81 +1,54 @@
 plugins {
-    id 'org.springframework.boot' version '2.5.1'
     id 'java'
+    id 'org.springframework.boot' version '2.5.5'
+    id 'io.spring.dependency-management' version '1.0.11.RELEASE'
 }
 
-apply plugin: 'io.spring.dependency-management'
-apply plugin: 'jacoco'
-
 group = 'uk.gov.digital.ho.hocs'
 version = '0.0.1-SNAPSHOT'
 sourceCompatibility = '11'
 
-configurations {
-    compileOnly {
-        extendsFrom annotationProcessor
-    }
-}
-
 repositories {
     mavenCentral()
+    maven { url 'https://jitpack.io' }
 }
 
-task copyLombok(type: Copy) {
-	outputs.upToDateWhen { false }
-	into "${buildDir}/libs"
-	include "lombok*.jar"
-	from configurations.compileClasspath
-}
-
-build.dependsOn(copyLombok)
-
-repositories {
-	mavenCentral()
-    maven { url 'https://jitpack.io' }
+ext {
+    springCloudVersion = "Hoxton.SR12"
 }
 
 dependencies {
-    implementation group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.12.5'
-    implementation group: 'com.fasterxml.jackson.core', name: 'jackson-core', version: '2.12.5'
-    implementation group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jsr310', version: '2.12.5'
-
-    implementation('org.springframework.boot:spring-boot-starter-web') {
```

Comment:
This is the one implementation I wouldn't change, tomcat at least used to be slower than undertow so probably want to keep this to keep it in line with the other repos...